pairselection.hxx

```diff
@@ -16,6 +16,14 @@
 
 namespace pairselection {
 
+    auto filterGoodParticles(auto df, const std::string& pairname, const std::string& filtername){
```

Comment:
I would suggest to rename it to `filterGoodPair`, since it could be misunderstood.

Another point for my understanding: For which purpose is `const std::string& filtername` needed? To name the filter for the report, right?

Reply:
yes its for the report - i will rename it